cli/admin/src/test/java/io/pravega/cli/admin/segmentstore/AbstractSegmentStoreCommandsTest.java

```diff
@@ -392,6 +392,13 @@ public void testEvictReadIndexCacheCommand() throws Exception {
         Assert.assertNotNull(EvictReadIndexCacheCommand.descriptor());
     }
 
+    @Test
+    public void testEvictReadIndexCacheForSegmentCommand() throws Exception {
+        String commonResult = TestUtils.executeCommand("storage evict-read-index-cache 0 testSegment localhost", STATE.get());
```

Comment:
if you change order between localhost testSegment like below it will fix problem for you.
`String commonResult = TestUtils.executeCommand("storage evict-read-index-cache 0 localhost testSegment", STATE.get());`


Also,  I think we should have segmentStoreHost as first parameter, containerId as second parmeter and then rest of the parameters with optional at the end.

Please do the same change for other 2 commands